src/crud/session.py

```
@@ -320,17 +393,83 @@ async def delete_session(
+            delete(models.SessionPeer).where(
+                models.SessionPeer.session_name == session_name,
+                models.SessionPeer.workspace_name == workspace_name,
+            )
+        )
+
+        # Finally, delete the session itself
+        await db.delete(honcho_session)
+        await db.commit()
+        logger.debug("Session %s and all associated data deleted", session_name)
+    except Exception as e:
+        logger.error("Failed to delete session %s: %s", session_name, e)
```

Ruff (0.14.7)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)